packages/plugin-stripe/src/hooks/deleteFromStripe.ts

```diff
@@ -3,7 +3,7 @@ import type { CollectionAfterDeleteHook, CollectionConfig } from 'payload/types'
 import { APIError } from 'payload/errors'
 import Stripe from 'stripe'
 
-import type { StripeConfig } from '../types.js'
+import type { PluginConfig } from '../types.js'
```

Comment:
Is `StripePluginConfig` a better choice here?

Reply:
All the other plugins were using `PluginConfig` as a standard. This was to make it consistent with them.

Reply:
Ok. IMO the more declarative the better but I'm good with it. When you go to import the type, for example, intelliscence is going to suggest types from unrelated pkgs, rather than the _one_ your looking for. There's also the problem with needing to alias imports if they are imported into the same file, because they now share the same name. Something to consider. I would be much more likely to type `StripePluginConf...` when looking for the type here.

Reply:
That is a valid point. During my initial review, I thought this was purely internal. I see it _was_ being exported in v2, but with the new pattern, has not been added to the `src/exports/types.ts` file. Let me push a change.

I will likely retroactively do this for the other plugins.